pkg/resources/schema_acceptance_test.go

```diff
@@ -192,55 +547,55 @@ func TestAcc_Schema_DefaultDataRetentionTime(t *testing.T) {
 				ConfigDirectory: acc.ConfigurationDirectory("TestAcc_Schema_DefaultDataRetentionTime/WithoutDataRetentionSet"),
 				ConfigVariables: configVariablesWithoutSchemaDataRetentionTime(5),
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr("snowflake_schema.test", "data_retention_days", r.IntDefaultString),
+					resource.TestCheckResourceAttr("snowflake_schema.test", "data_retention_time_in_days", r.IntDefaultString),
```

Comment:
this rename is a breaking change and should be communicated in the migration guide

Reply:
Also should be handled in the state upgrader